src/team-extension.ts

```diff
@@ -317,11 +330,10 @@ export class TeamExtension  {
     }
 
     private ensureInitialized(): boolean {
-        if (!this._gitContext
-                || !this._gitContext.RemoteUrl
+        if (!this._repoContext
                 || !this._serverContext
                 || !this._serverContext.RepoInfo.IsTeamFoundation) {
-            this.setErrorStatus(Strings.NoGitRepoInformation);
+            this.setErrorStatus(Strings.NoRepoInformation);
             return false;
```

Comment:
I took a note for this.  It may make sense to indicate the type of repo a particular command applies to.